server/sonar-server/src/main/java/org/sonar/server/app/EmbeddedTomcat.java

```diff
@@ -61,6 +61,7 @@ void start() {
       tomcat.start();
       new TomcatStartupLogs(Loggers.get(getClass())).log(tomcat);
     } catch (LifecycleException e) {
+      e.printStackTrace();
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Use a logger to log this exception. ([squid:S1148](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1148))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fdr%2Fgradle&issues=AWHZGQ8kaf12pdu-6k13&open=AWHZGQ8kaf12pdu-6k13)

[ISSUE_KEY]: AWHZGQ8kaf12pdu-6k13